### ISSUE TITLE
Missing close button aria-label

### ISSUE THREAD

**Issue by pkayokay** [mapbox/mapbox-gl-geocoder#182]
<img width="452" alt="screen shot 2019-02-01 at 11 15 40 am" src="https://user-images.githubusercontent.com/13375749/52135146-221beb80-2613-11e9-89f7-480ff3ec9b33.png">

Though it is on here.
https://github.com/mapbox/mapbox-gl-geocoder/blob/master/lib/index.js#L99

**Comment by andrewharvey** [comment]
Sorry I can't seem to replicate this, :confused: 

![screenshot from 2019-02-02 10-20-08](https://user-images.githubusercontent.com/117278/52154900-278d3a80-26d4-11e9-9d3d-8023cabbe429.png)

**Comment by pkayokay** [comment]
Using `mapbox-gl-geocoder": "^2.0.1"`

Tested on the following browsers:
- Brave (0.58.21)
- Chrome (72.0.3626.81)
- FireFox Quantum 65.0
- Safari (12.0.2)

Strange. It might just be an implementation issue on our Rails/React app, still investigating further.

**Comment by andrewharvey** [comment]
That feature was only added in 2.1.0

https://github.com/mapbox/mapbox-gl-geocoder/commit/f5aa2985c5186a43659f69a1aaab9b020bccd7f2

Which exact version is being installed by npm/yarn?

**Comment by andrewharvey** [comment]
I haven't heard back so I'll close this since it appears to be caused by the old version you're using. If you can replicate in the latest version, we can re-open.
